packages/components/src/circular-option-picker/index.js

```diff
@@ -64,6 +64,7 @@ function DropdownLinkAction( {
 			renderToggle={ ( { isOpen, onToggle } ) => (
 				<Button
 					aria-expanded={ isOpen }
+					aria-haspopup="true"
```

Comment:
I've chosen `true` as value instead of a more concrete one because 1) it's still valid as of ARIA 1.1 and 2) all the cases in which `aria-haspopup` is present we used this value so far. Happy to set it to a different value.